### PR TITLE
dijkstra shortest path is working again

### PR DESCRIPTION
**Pull request by ldgeo**
Now takes referentials as inputs and returns transformations as paths

### CODE REVIEW

File: python/pg_li3ds/__init__.py [hunk 1]
```diff
@@ -13,6 +15,105 @@
...
+            plpy.notice("No path from ref:{} to ref:{} with config {}"
+                        .format(source, target, config))
+            return []
+        shortest_path.insert(0, x)
+
+    # we have referentials now we need all transformations
+    # assembling refs by pair
+    ref_pair = [
+        shortest_path[i:i + 2]
+        for i in xrange(0, len(shortest_path) - 1)]
```
Comment (elemoine) [inline python/pg_li3ds/__init__.py]: Super minor, but let's use `range` for py3 compat. There shouldn't be any performance difference in this case.

File: python/pg_li3ds/__init__.py [hunk 2]
```diff
@@ -13,6 +15,105 @@
 }
 
 
+def dijkstra(config, source, target):
+    '''
+    returns the transfo lisarray[ti.id] <@ array[1, 2, 3, 4, 5, 6, 7, 8]t needed to go from source referential to target
+    referential
```
Comment (elemoine) [inline python/pg_li3ds/__init__.py]: Can you please make this shorter for pep8?

File: python/pg_li3ds/__init__.py [hunk 3]
```diff
@@ -13,6 +15,105 @@
...
+        raise Exception("No referential with id {}".format(source))
+    if target not in graph:
+        raise Exception("No referential with id {}".format(target))
+
+    M = set()
+    d = {source: 0}
+    p = {}
+    next_nodes = [(0, source)]
+
+    while next_nodes != []:
```
Comment (elemoine) [inline python/pg_li3ds/__init__.py]: Minor, but can't you avoid creating an object (`[]`) by using `while len(next_nodes)`?

File: python/pg_li3ds/__init__.py (hunk 2 above)
Comment (ldgeo) [inline python/pg_li3ds/__init__.py]: was a bad copy/paste. Done

File: python/pg_li3ds/__init__.py (hunk 1 above)
Comment (ldgeo) [inline python/pg_li3ds/__init__.py]: done

**Pull request merged**